src/Windowing/Silk.NET.Windowing.Common/Structs/WindowOptions.cs

```diff
@@ -120,26 +121,63 @@ public WindowOptions
             VSync = isVSync;
         }
 
+        private static Lazy<WindowOptions> _default = new Lazy<WindowOptions>
+        (
+            () => new WindowOptions
+            (
+                true, new Point(50, 50), new Size(1280, 720), 0.0, 0.0, GraphicsAPI.Default,
+                Assembly.GetEntryAssembly()?.GetName().Name ?? "Silk.NET Window", WindowState.Normal,
+                WindowBorder.Resizable, true, true, VideoMode.Default
+            )
+        );
+        
+        private static Lazy<WindowOptions> _vulkanDefault = new Lazy<WindowOptions>(
+            () => new WindowOptions
+            (
+                true, new Point(50, 50),
+                new Size(1280, 720), 0.0, 0.0, GraphicsAPI.DefaultVulkan,
+                Assembly.GetEntryAssembly()?.GetName().Name ?? "Silk.NET Window", WindowState.Normal,
+                WindowBorder.Resizable, false, false, VideoMode.Default
+            ));
+
+        private static Lazy<WindowOptions> _aotDefault = new Lazy<WindowOptions>
```

Comment:
Separate AOT defaults likely aren't needed. If CoreRT reflection is disabled, GetEntryAssembly will return null, thus will default to Silk.NET Window on the regular properties.

Reply:
GetEntryAssembly & friends throw in CoreRT, so the static constructor of WindowOptions throws if this is not lazy.
I can remove the AOT defaults.